BlazarUI/app/scripts/components/settings/SettingsContainer.jsx

```diff
@@ -54,7 +54,7 @@ class SettingsContainer extends Component {
     return (
       <Headline>
         <Icon type="fa" name="wrench" classNames="headline-icon" />
-        <span>Settings</span>
+        <span>Settings</span> <br />
```

Comment:
Not a huge deal, but consider using css to force the newline instead of using the html `<br />`